facebook/__init__.py

```diff
@@ -163,9 +163,12 @@ def put_object(self, parent_object, connection_name, **data):
 
         """
         assert self.access_token, "Write operations require an access token"
+        file = None if file is None else {"file": file}
```

Comment:
Compound statements like this are hard to read. It is difficult to tell at a glance what this is doing, especially since `file` is being reassigned. Something like:

     if file is None:
          files = None
     else:
          files = {"file": file}

would be a lot more readable. A comment explaining why/when this is necessary would also be great.